### PR TITLE
rebar.config tidying up

### PR DESCRIPTION
**Pull request by rustaem**
Self explanatory

### CODE REVIEW

**Comment by cytadela8** [comment]
@rustaem `===> Application needed for release not found: aehyperchains`

File: rebar.config [hunk 1]
```diff
@@ -20,234 +20,276 @@
...
+		{ref, "66e5a51"}}},
+
+	{mnesia_leveled, {git, "https://github.com/aeternity/mnesia_leveled.git",
+		{ref, "11c480b"}}},
+
+	{aeminer, {git, "https://github.com/aeternity/aeminer.git",
+		{ref, "0a82f0f"}}},
+
+	{aebytecode, {git, "https://github.com/aeternity/aebytecode.git",
+		{ref, "951db9f"}}},
```
Comment (cytadela8) [inline rebar.config:72]: You should not change any refs!

File: rebar.config [hunk 2]
```diff
@@ -20,234 +20,276 @@
...
+	{trace_runner, {git, "git://github.com/uwiger/trace_runner.git",
+		{ref, "69dc388"}}},
+
+	% for hyperchains and the testing suite
+	{aesophia_aci_encoder, {git, "https://github.com/aeternity/aesophia_aci_encoder",
+		{ref, "41d0427"}}},
+
+	% For automatic building of the StakingContract.json
+	% Do NOT use 'tag' or 'ref' clause here as the source code of the contract is subject to change.
+	{hyperchains_contract, {git, "git://github.com/aeternity/hyperchains-contract.git",
```
Comment (cytadela8) [inline rebar.config:112]: This shoudn't be on master

**Review by cytadela8 — CHANGES_REQUESTED** [review]

File: rebar.config (hunk 1 above)
Comment (cytadela8) [reply rebar.config:72]: (accidently :P)

**Review by cytadela8 — COMMENTED** [review]
It looks good now. I would like to leave it to others if we want to introduce those style changes.

**Review by cytadela8 — APPROVED** [review]

**Review by hanssv — COMMENTED** [review]
Unless rebar3 starts to require TABs in the config I don't think we should have them...

File: rebar.config [hunk 3]
```diff
@@ -20,234 +20,271 @@
...
-
-        % upstream is not maintained anymore
-        {sha3, {git, "https://github.com/aeternity/erlang-sha3",
-               {ref, "b5f27a2"}}},
-
-        % for lightweight trace-based debugging
-        {trace_runner, {git, "git://github.com/uwiger/trace_runner.git",
-                        {ref, "69dc388"}}},
+	%% off-the-shelf depdendencies
+	{enoise, {git, "https://github.com/aeternity/enoise.git", {ref, "1e6ee67"}}},
```
Comment (hanssv) [inline rebar.config:36]: If tidying up, this one isn't off-the shelf 😅

**Review by hanssv — CHANGES_REQUESTED** [review]
Looks tidy, but no tabs please.

**Comment by rustaem** [comment]
> Unless rebar3 starts to require TABs in the config I don't think we should have them...

You know it's one stroke in any IDE, right?.. :) Changed anyway.

File: rebar.config (hunk 3 above)
Comment (rustaem) [reply rebar.config]: I wouldn't want to cut any pre-existing comments -- there is a reason for their existence probably. Are you certain it must go?

Comment (hanssv) [reply rebar.config]: No, I mean the dependency should be moved further down, to a group where it belongs.

**Comment by hanssv** [comment]
> > Unless rebar3 starts to require TABs in the config I don't think we should have them...
> 
> You know it's one stroke in any IDE, right?.. :) Changed anyway.

Yes, and any sane editor will replace that key-stroke with the right amount of white space.

**Comment by rustaem** [comment]
> > > Unless rebar3 starts to require TABs in the config I don't think we should have them...
> > 
> > 
> > You know it's one stroke in any IDE, right?.. :) Changed anyway.
> 
> Yes, and any sane editor will replace that key-stroke with the right amount of white space.

No, I mean it's a single key-stroke to change tabs to spaces or vv :)
And the dep is moved.

**Review by hanssv — APPROVED** [review]

**Review by cytadela8 — APPROVED** [review]

**Review by cytadela8 — APPROVED** [review]

**Review by radrow — APPROVED** [review]

**Pull request closed without merging**